sycl/source/detail/graph_impl.hpp

```diff
@@ -1329,6 +1329,9 @@ class exec_graph_impl {
   /// host_task.
   void makePartitions();
 
+  /// Proceeds to graph fusion
```

Comment:
This comment could be more descriptive, doesn't need to be super detailed but could at least name the layered extension